src/mpid/ch4/src/ch4r_callbacks.c

```diff
@@ -548,6 +568,7 @@ int MPIDIG_send_long_req_target_msg_cb(int handler_id, void *am_hdr, void **data
     MPIR_FUNC_VERBOSE_ENTER(MPID_STATE_MPIDIG_SEND_LONG_REQ_TARGET_MSG_CB);
 
     root_comm = MPIDIG_context_id_to_comm(hdr->context_id);
+  root_comm_retry:
```

Comment:
@hajimefu maybe we want to put this label after the `if` check like the way it is done above?

Reply:
👍 Good catch.